### PR TITLE
Dashboard: Edit button, text-changes, buttons

### PR DESCRIPTION
**Pull request by 0-1123**

### CODE REVIEW

File: app/assets/stylesheets/components/_card.scss [hunk 1]
```diff
@@ -17,6 +17,7 @@
   font-weight: bold;
   background: none;
   color: rgba(0,0,0,0.8);
+  padding-left: 8px;
```
Comment (doersam) [inline app/assets/stylesheets/components/_card.scss]: this padding is probably going to affect the home page. can you check this ?

File: app/views/bookings/_my_spaces.html.erb [hunk 2]
```diff
@@ -20,7 +20,9 @@
             <h5 class="card-title" style="margin: 0; padding: 0;"><%= space.address %></h5>
             <h5 class="card-text" style="margin: 0; padding: 0;">Price: <%= space.price %> euro</h5>
             <h5 class="card-text mb-2" style="margin: 0; padding: 0;">Size: <%= space.size %> m^2</h5>
-            <%= link_to "Edit", space_path(space), class: "button h5" %>
+            <% if space.bookings.empty? %>
+            <%= link_to "Edit", edit_space_path(space), class: "button h5" %>
```
Comment (doersam) [inline app/views/bookings/_my_spaces.html.erb]: why is the btn having a class h5 ?

Comment (0-1123) [reply app/views/bookings/_my_spaces.html.erb]: The h5 was already there, idk

File: app/views/bookings/index.html.erb [hunk 3]
```diff
@@ -24,9 +24,9 @@
       <%= render "past_bookings"  %>
     </div>
     <div class="tab-pane fade" id="manage_my_spaces" role="tabpanel" aria-labelledby="manage_my_spaces-tab">
-        <h1>
+        <div>
```
Comment (doersam) [inline app/views/bookings/index.html.erb]: this solved a lot of problems that I could not find

**Pull request merged**